src/frontend/src/pages/part/PartDetail.tsx

```diff
@@ -341,6 +352,7 @@ export default function PartDetail() {
         label: t`Price Range`,
         value_formatter: () => {
           const { data } = useSuspenseQuery({
+            refetchOnWindowFocus: false,
```

Comment:
C (non blocking): This appears multiple times, should this maybe be a default passed to the queryclient?

Reply:
Good suggestion, I have added a default value to the query manager